nixos/modules/services/databases/postgresql.nix

```diff
@@ -293,7 +293,8 @@ in
       # Note: when changing the default, make it conditional on
       # ‘system.stateVersion’ to maintain compatibility with existing
       # systems!
-      mkDefault (if versionAtLeast config.system.stateVersion "20.03" then pkgs.postgresql_11
+      mkDefault (if versionAtLeast config.system.stateVersion "21.11" then pkgs.postgresql_13
+            else if versionAtLeast config.system.stateVersion "20.03" then pkgs.postgresql_11
```

Comment:
befor 20.03 is deprecated, how about we remove the else if at the bottom as well ?(with 17.09)

Reply:
I don't think we should remove any of these, because it would upgrade installations with an old stateVersion.